src/io/jesi/backpack/random.cljc

```diff
@@ -55,7 +55,7 @@
     (apply str)))
 
 (defn- gen-str [chars size]
-  (apply str (take size (repeatedly #(rand-nth chars)))))
+  (clojure.string/join (repeatedly size (fn* [] (rand-nth chars)))))
```

Comment:
Why use special form `fn*`?
(I'm getting deja vu, I think I may have asked this question in the past but forgot the rationale)

Reply:
I think that was kibit. I'll revert